src/components/Svg/Svg.tsx

```diff
@@ -0,0 +1,59 @@
+import { PropsWithChildren, SVGProps } from 'react';
+
+import { ClassNameType } from '@/types/util';
+import { tw } from '@/utils/tailwind.util';
+
+type SvgProps = SVGProps<SVGSVGElement> & {
+  /**
+   * width, height와 동시에 적용하는 size입니다. (단위 px)
+   * @default `24`
+   */
+  size?: number;
+
+  /**
+   * size가 존재하더라도 무시하고 적용되는 width입니다. (단위 px)
+   * @default `24`
+   */
+  width?: number;
+
+  /**
+   * height 존재하더라도 무시하고 적용되는 width입니다. (단위 px)
+   * @default `24`
+   */
+  height?: number;
+
+  /**
+   * tailwind으로 svg css 수정할 수 있어요! (크기 조정도 가능합니다)
+   * https://tailwindcss.com/docs/fill
+   * https://tailwindcss.com/docs/stroke
+   * https://tailwindcss.com/docs/stroke-width
+   */
+  className?: ClassNameType;
+};
+
+export const Svg = ({
+  size,
+  width,
+  height,
+  children,
+  className,
+  viewBox,
+  ...rest
+}: PropsWithChildren<SvgProps>) => {
+  const w = width ?? size ?? 24;
+  const h = height ?? size ?? 24;
+  const vb = viewBox ?? `0 0 24 24`;
```

Comment:
viewbox와 width, height을 각각 다르게 가져가도 되는지 궁금합니다!

Reply:
가능합니다! 하지만 svg 특성상 비율이 깨져보일 수도 있기 때문에 `viewBox와 width, height를 통일해서 사용하는 방법`을 추천해요~

width, height가 svg 컴포넌트가 렌더링될 대`진짜 크기(px, rem)`을 나타낸다면, viewBox는 svg 컴포넌트에서 하위 요소(path, circle 등등)의 크기 `비율`이라고 알고 있어요.  예를 들어, 
-  width, height > viewbox :  지정한 범위를 넘어서 svg가 짤려서 보임
-  width, height < viewbox : 지정한 범위보다 svg보다 작게 보임

아래 블로그글을 참고해서 구현한 부분이라 자세한 설명은 아래 참고글을 보시면 좋을 것 같아요

- [Tecoble - SVG viewBox를 알아보자](https://tecoble.techcourse.co.kr/post/2021-10-24-svg-viewBox/)

Reply:
width, height,  viewBox를 따로따로 가져가기보다 아래와 같이 맞추는 것은 어떨까요?
Svg 컴포넌트를 사용하는 개발자가 viewBox까지는 신경을 못 쓸 것 같습니다.
```suggestion
  const w = width ?? size ?? 24;
  const h = height ?? size ?? 24;
  const vb = `0 0 ${w || 24} ${h || 24}`;
```

Reply:
넵 수정하는대로 머지할게요!